#######################
### Controller      ###
### (20/7), (5.8/3) ###
#######################
gain_lambda:
  p3d: 20
  yolo: 7
vw_limit:               # units: [rad/s]
  max: 2.09
  min: 0.017

#########################
### Image Recognition ###
#########################
target: "car"
probability: 0.05

####################
### Vehicle info ###
####################
platform: 2             # Select uav type (1: typhoon, 2: miniyy, 3:techpod)

typhoon:              
  intrinsic_matrix:
    FL_1x: 1280.784     # units: [pixel]
  relPos_panPlane:      # frame:  FLU (uav body); units: [m]
    x: -0.026
    y: 0.000
    z: -0.1
  relPos_camPan:        # frame: gimbal pan
    x: 0.016
    y: 0.000
    z: 0.062

miniyy:                 # horizontal_fov: defined in uav sdf file
  intrinsic_matrix:
    FL_1x: 1280.784     
  relPos_panPlane:      # flu
    x: 1.156
    y: 0.000
    z: 0.062
  relPos_camPan:        # pan
    x: 0.016
    y: 0.000
    z: 0.062

techpod:
  intrinsic_matrix:
    FL_1x: 1280.784
  relPos_panPlane:      # flu
    x: 0.356
    y: 0.000
    z: 0.002
  relPos_camPan:        # pan
    x: 0.016
    y: 0.000
    z: 0.062

############################
### User Function Select ###
############################
user: 1                 # (0: key insert, 1: topic)